www/css/style.css

```diff
@@ -0,0 +1,250 @@
+/* Empty. Add your own CSS if you like */
```

Comment:
I really have never coded with CSS before and I really need to learn how to. HTML looks gross if you don't know what you're doing. Could you split this into a few more files with descriptive names? I think that might make it more readable. It's a bit long as it is.